DEVELOPMENT.md

```diff
@@ -49,11 +56,7 @@ In addition to the REPL development inside a single module, it's possible to run
 
 ## Style Guide
 
-The Clojure code in this project follows the [Clojure Style Guide][2].
-
-### Exceptions:
-
-* we use two empty lines between top-level forms
+The Clojure code in this project mainly follows the [Clojure Style Guide][2], except that we use 2 (two) empty lines between top-level forms.
```

Comment:
We no longer use two empty lines between functions. In fact we use `cljfmt` now. So it would be good to mention `cljfmt` here.

Reply:
What about this?

"The Clojure code in this project mainly follows the [Clojure Style Guide][2], enforced by `cljfmt`. For more details, please check `cljfmt.edn`."

Reply:
Perfect